### DISCUSSION TITLE
--export envvars can create ambiguity

### DISCUSSION THREAD

**Discussion by jeffmacdonald** [cloudreach/sceptre#835] [Ideas]
This is in Sceptre v2.

Lets say you have 2 stacks that each have an output named FooBarOutput , and somehow they are dependent upon each other. (In mycase.. I have 2 stacks that create RDS instances and both output "EndPointUrl")

If you run: sceptre list outputs my/stack/name/foo.yaml --export envvar

You'll get

export SCEPTRE_FooBarOutput=_some thing_

But it will be unclear which template/stack that came from. 

Proposal

export SCEPTRE_$template_file_name_FooBarOutput=_some_thing_

**Comment by ngfgrant** [comment]
Hey @jeffmacdonald 

You shouldn't be able to have two stacks that are dependent on one another as this would be a cyclical dependency and sceptre won't allow it.

I get the point but wonder if it needs a little more thought? This would also potentially break things for others so wouldn't be a v2 thing.
